src/rdkafka_admin.h

```diff
@@ -430,6 +430,28 @@ struct rd_kafka_DeleteAcls_result_response_s {
         rd_list_t matching_acls; /**< Type (rd_kafka_AclBinding_t *) */
 };
 
+/**
+ * @name ElectLeaders
+ */
+
+/**
+ * @struct ElectLeaders request object
+ */
+struct rd_kafka_ElectLeaders_s {
+        rd_kafka_ElectionType_t electionType; /*Election Type*/
+        rd_kafka_topic_partition_list_t
+            *partitions; /*TopicPartitions for election*/
+};
+
+
+/**
+ * @struct ElectLeaders result object
+ */
+struct rd_kafka_ElectLeadersResult_s {
+        rd_kafka_resp_err_t error_code;
```

Comment:
Use `err` instead to follow the current convention. No need to change the public function.